runtime/gc_glue_java/ArrayletObjectModel.hpp

```diff
@@ -864,15 +864,15 @@ class GC_ArrayletObjectModel : public GC_ArrayletObjectModelBase
 	dataAddrSlotForDiscontiguous(J9IndexableObject *arrayPtr)
 	{
 		/* If double mapping is enabled only, arraylet will have a discontiguous layout.
-		 * If sparse-heap is enabled, arraylet will have a contiguous layout. For now
-		 * we can't simply Assert only the discontiguous case because there could also
-		 * exist hybrid arraylets (which will be dicontinued in the future) */
+		* If sparse-heap is enabled, arraylet will have a contiguous layout. For now
+		* we can't simply Assert only the discontiguous case because there could also
+		* exist hybrid arraylets (which will be dicontinued in the future) */
```

Comment:
These lines were indented correctly before; please restore the space before each `*`.